randovania/game_description/requirements/resource_requirement.py

```diff
@@ -20,15 +19,27 @@
     from randovania.game_description.resources.simple_resource_info import SimpleResourceInfo
 
 
-@dataclasses.dataclass(frozen=True, slots=True)
```

Comment:
why remove the dataclass? does removing the frozen aspect affect anything?

Reply:
mypyc doesn't like this much :(